buildh/writers.py

```diff
@@ -127,9 +126,8 @@ def write_OP_alternate(fileout, universe_woH, dic_OP, dic_lipid):
         This is the universe *without* hydrogen.
     dic_OP : ordered dictionnary
         Each key of this dict is a couple carbon/H with the OP values as a list.
-    dic_lipid : dictionnary
-        Comes from dic_lipids.py. Contains carbon names and helper names needed
-        for reconstructing hydrogens.
+    resname : str
+        lipid name
```

Comment:
We agree that we are talking about the residue name in the PDB file, right? If yes, maybe we should say it here?

Reply:
sure.

Reply:
No, in fact, it's the residue name from the dictionary from the json.

Reply:
OK, so maybe you can lipid residue name in the json file?

Reply:
ok